src/chart/rendering/drawinterlacing.h

```diff
@@ -9,7 +9,10 @@ namespace Vizzu::Draw
 class DrawInterlacing : private DrawingContext
 {
 public:
-	DrawInterlacing(const DrawingContext &context, bool text);
+	explicit DrawInterlacing(const DrawingContext &context);
+
+	void drawGeometries();
+	void drawTexts();
```

Comment:
This function will be not part of the "interlacing" but another Draw class (DrawMarkerLabels). I didn't use the 'drawLabels' name on purpose, because it is called from `DrawAxes::drawLabels()` (the same name function). 

This will be changed in the next PR